snap/snapcraft.yaml

```diff
@@ -0,0 +1,94 @@
+name: jitsi
+version: "1.0"
+summary: Jitsi
+description: |
+  This is a snap of Jitsi
+
+grade: stable
+confinement: strict
+base: core18
+
+# might need certificate configuration
+
+parts:
+  jicofo:
+    plugin: dump
+    source: https://download.jitsi.org/jitsi/debian/jicofo_1.0-508-1_amd64.deb
+  jitsi-meet-prosody:
+    plugin: dump
+    source: https://download.jitsi.org/jitsi/debian/jitsi-meet-prosody_1.0.3729-1_all.deb
+  jitsi-meet-web:
+    plugin: dump
+    source: https://download.jitsi.org/jitsi/debian/jitsi-meet-web_1.0.3729-1_all.deb
+  jitsi-videobridge:
+    plugin: dump
+    source: https://download.jitsi.org/jitsi/debian/jitsi-videobridge_1126-1_amd64.deb
+  jitsi-meet-web-config:
+    plugin: dump
+    source: https://download.jitsi.org/jitsi/debian/jitsi-meet-web-config_1.0.3729-1_all.deb
+    override-pull: |
+      snapcraftctl pull
+      echo "jitsi" > etc/hostname
+      echo "127.0.0.1 localhost jitsi" > etc/hosts
+      #usr/share/jitsi-meet/scripts/install-letsencrypt-cert.sh
+
+    stage-packages:
+      - prosody
+      - lua-bitop
+      - lua-event
+      - lua-expat
+      - lua-filesystem
+      - lua-sec
+      - lua-socket
+      - lua5.2
+      - openjdk-8-jre-headless
+      - ca-certificates-java
+      - java-common
+
+  jitsi-scripts:
+    source: ./scripts
+    plugin: dump
+
+layout:
+  /etc/jitsi:
+    bind: $SNAP/etc/jitsi
+
+environment:
+  JVB_HOSTNAME: jitsi
+  JVB_PORT: 5347
+  JVB_SECRET:
+  JVB_OPTS: --apis=rest,xmpp
+  JAVA_SYS_PROPS: -Dnet.java.sip.communicator.SC_HOME_DIR_LOCATION=/etc/jitsi -Dnet.java.sip.communicator.SC_HOME_DIR_NAME=videobridge -Dnet.java.sip.communicator.SC_LOG_DIR_LOCATION=$HOME/jitsi -Djava.util.logging.config.file=/etc/jitsi/videobridge/logging.properties
+  JITSI_LOGFILE: $HOME/jitsi/jvb.log
+  JICOFO_LOGFILE: $HOME/jitsi/jicofo.log
+  JICOFO_HOST: localhost
+  JICOFO_HOSTNAME: jitsi
+  JICOFO_PORT: 5347
+  JICOFO_SECRET:
+  JICOFO_AUTH_USER: focus
+  JICOFO_AUTH_DOMAIN: auth.virt
+  JICOFO_AUTH_PASSWORD:
+
+apps:
+  jitsi-videobridge:
+    command: jitsi-videobridge.sh
+    daemon: simple
+    restart-condition: on-abnormal
+    plugs:
+      - home
+      - network
+      - network-bind
+    environment:
+      JAVA_HOME: $SNAP/usr/lib/jvm/java-11-openjdk-amd64
```

Comment:
@damencho Is using Java 11 OK now?

Reply:
Should be fine, although jvb2 is optimized for java8 ...